pkgs/top-level/all-packages.nix

```diff
@@ -19798,9 +19798,7 @@ in
 
   mediainfo-gui = callPackage ../applications/misc/mediainfo-gui { };
 
-  # mediathekview needs JavaFX, which currently only is available inside OracleJRE
-  # we might be able to get rid of it, as soon as we have an OpenJRE with OpenJFX included
-  mediathekview = callPackage ../applications/video/mediathekview { jre = oraclejre; };
+  mediathekview = callPackage ../applications/video/mediathekview { jre = openjdk11; };
```

Comment:
Previously it pulled only a simple JRE, now it's pulling a full JDK. I suppose this is the main reason for the big closure size increase.
I don't know if there is a way to use only the JRE from `openjdk11`. Or maybe it could pull `adoptopenjdk-jre-bin` instead?

Reply:
I'll look into it, put I thought that the default JRE is the OpenJDK...

Reply:
openjdk11.jre doesn't seem to exist. adoptopenjdk-jre-bin throws an exeception:

````
./result/bin/mediathek
. Portable Mode: false
. Programmstart: 15.11.2019 01:24:57
. Version: 13.5.0
. Verzeichnis Einstellungen: /home/addy/.mediathek3
Loading library prism_es2 from resource failed: java.lang.UnsatisfiedLinkError: /home/addy/.openjfx/cache/13/libprism_es2.so: libGL.so.1: cannot open shared object file: No such file or directory
java.lang.UnsatisfiedLinkError: /home/addy/.openjfx/cache/13/libprism_es2.so: libGL.so.1: cannot open shared object file: No such file or directory
	at java.base/java.lang.ClassLoader$NativeLibrary.load0(Native Method)
	at java.base/java.lang.ClassLoader$NativeLibrary.load(Unknown Source)
	at java.base/java.lang.ClassLoader$NativeLibrary.loadLibrary(Unknown Source)
	at java.base/java.lang.ClassLoader.loadLibrary0(Unknown Source)
	at java.base/java.lang.ClassLoader.loadLibrary(Unknown Source)
	at java.base/java.lang.Runtime.load0(Unknown Source)
	at java.base/java.lang.System.load(Unknown Source)
	at com.sun.glass.utils.NativeLibLoader.installLibraryFromResource(NativeLibLoader.java:214)
	at com.sun.glass.utils.NativeLibLoader.loadLibraryFromResource(NativeLibLoader.java:194)
	at com.sun.glass.utils.NativeLibLoader.loadLibraryInternal(NativeLibLoader.java:135)
	at com.sun.glass.utils.NativeLibLoader.loadLibrary(NativeLibLoader.java:53)
	at com.sun.prism.es2.ES2Pipeline.lambda$static$0(ES2Pipeline.java:68)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.prism.es2.ES2Pipeline.<clinit>(ES2Pipeline.java:50)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Unknown Source)
	at com.sun.prism.GraphicsPipeline.createPipeline(GraphicsPipeline.java:218)
	at com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.init(QuantumRenderer.java:91)
	at com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.run(QuantumRenderer.java:124)
	at java.base/java.lang.Thread.run(Unknown Source)
Exception in thread "main" java.lang.UnsupportedOperationException: Internal Error
	at com.sun.glass.ui.gtk.GtkApplication.lambda$new$6(GtkApplication.java:189)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.<init>(GtkApplication.java:171)
	at com.sun.glass.ui.gtk.GtkPlatformFactory.createApplication(GtkPlatformFactory.java:41)
	at com.sun.glass.ui.Application.run(Application.java:144)
	at com.sun.javafx.tk.quantum.QuantumToolkit.startup(QuantumToolkit.java:280)
	at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:269)
	at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:158)
	at javafx.embed.swing.JFXPanel.initFx(JFXPanel.java:241)
	at javafx.embed.swing.JFXPanel.<init>(JFXPanel.java:257)
	at mediathek.Main.initializeJavaFX(Main.java:302)
	at mediathek.Main.main(Main.java:272)
````

Reply:
Hum yeah it needs JavaFX, apparently it's [not part of AdoptOpenJDK build yet](https://adoptopenjdk.net/migration.html#openjfx) ... :/

So yes only our OpenJDK ships with OpenJFX since #63574
And we don't have a simple JRE with OpenJFX.